app/models/concerns/findings/process_email.rb

```diff
@@ -0,0 +1,79 @@
+module Findings::ProcessEmail
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def receive_mail mail
+      finding_id = extract_finding_id mail
+      user       = find_user(mail, finding_id) if finding_id
+      finding    = set_finding(finding_id, user) if user
+
+      if finding
+        finding.finding_answers.create generate_finding_answer_from_mail(mail, user)
```

Comment:
```suggestion
        finding.finding_answers.create! generate_finding_answer_from_mail(mail, user)
```
Lo haría con el bang así nos enteramos si falla, sino va a pasar "en silencio".

Reply:
perfecto